x/oracle/keeper/msg_server.go

```diff
@@ -52,21 +50,8 @@ func (ms msgServer) AggregateExchangeRatePrevote(
 	}
 
 	aggregatePrevote := types.NewAggregateExchangeRatePrevote(voteHash, valAddr, uint64(ctx.BlockHeight()))
-
 	ms.SetAggregateExchangeRatePrevote(ctx, valAddr, aggregatePrevote)
 
-	ctx.EventManager().EmitEvents(sdk.Events{
```

Comment:
Why do we not throw this event anymore?

Reply:
Because this events doesn't bring anything.